Dockerfile

```diff
@@ -1,22 +1,21 @@
 # Build image
```

Comment:
Dockerfile looks a lot nicer!

Reply:
these are working for me

mverrall@ ~/Documents/source/flyte (feature/update_flyte_to_use_go_modules) $ go test ./... -tags="integration acceptance"
ok      github.com/HotelsDotCom/flyte   (cached)
ok      github.com/HotelsDotCom/flyte/acceptancetest    6.770s
?       github.com/HotelsDotCom/flyte/acceptancetest/server     [no test files]
ok      github.com/HotelsDotCom/flyte/acceptancetest/urlutil    (cached)
ok      github.com/HotelsDotCom/flyte/audit     (cached)
ok      github.com/HotelsDotCom/flyte/auth      (cached)
ok      github.com/HotelsDotCom/flyte/collections       (cached)
ok      github.com/HotelsDotCom/flyte/datastore (cached)
ok      github.com/HotelsDotCom/flyte/execution (cached)
ok      github.com/HotelsDotCom/flyte/flow      (cached)
?       github.com/HotelsDotCom/flyte/flytepath [no test files]
ok      github.com/HotelsDotCom/flyte/httputil  (cached)
ok      github.com/HotelsDotCom/flyte/info      (cached)
ok      github.com/HotelsDotCom/flyte/json      (cached)
ok      github.com/HotelsDotCom/flyte/mongo     (cached)
?       github.com/HotelsDotCom/flyte/mongo/mongotest   [no test files]
ok      github.com/HotelsDotCom/flyte/pack      (cached)
ok      github.com/HotelsDotCom/flyte/server    (cached)
ok      github.com/HotelsDotCom/flyte/template  (cached)